src/main/kotlin/uk/gov/dluhc/emsintegrationapi/database/entity/RejectedReasonItem.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dluhc.emsintegrationapi.database.entity
+
+import javax.persistence.Column
+import javax.persistence.Embeddable
+import javax.validation.constraints.Size
+
+@Embeddable
+class RejectedReasonItem(
+    @field:Size(max = 100)
+    val electorReason: String?,
+
+    @field:Size(max = 50)
+    @Column(name = "rejection_reasons", length = 100)
+    val type: String?,

```

Comment:
This can probably afford to be an enum value (the only place we care about not using an enum is when sending to the external EMS in the `PostalVote` or `ProxyVote` objects)